test/LocalDataSource.spec.tsx

```diff
@@ -461,4 +462,15 @@ describe('LocalDataSource', () => {
     });
   });
 
+  describe('When column has aggregate function', () => {
+    it('should return the average of the \'Amount\' column', (done) => {
+      const dataSource = new LocalDataSource(localData, validColumnsSampleAggAverage);
+
+      dataSource.getAllRecords(10, 0, '').then((response: GridResponse) => {
+        expect(response.Aggregate.Amount).to.be.equal(157.36363636363637);
```

Comment:
Please review .closeTo or .within because you're dealing with floating point numbers.

Reply:
You're right